recipes/tests.rb

```diff
@@ -341,8 +341,17 @@ module load intelmpi && mpirun --help | grep '#{node['cfncluster']['intelmpi']['
 end
 
 ###################
-# EFA - GDR (GPUDirect RDMA)
+# EFA
 ###################
+bash 'check efa installed' do
+  cwd Chef::Config[:file_cache_path]
+  code <<-EFA
+      set -ex
+      modinfo efa
+      grep "EFA installer version: #{node['cfncluster']['efa']['installer_version']}" /opt/amazon/efa_installed_packages
```

Comment:
I think the test will fail if the existing EFA version mismatches `node['cfncluster']['efa']['installer_version']`, we can verify it in custom ami kitchen test.

Reply:
I believe you are right. do we have a variable in kitchen telling us whether the test is for a custom ami or not because ideally we want to do the following:
1. for official AMIs verify that EFA is installed and matches the expected version
2. for custom AMIs only verify that EFA is installed by skip version check

Reply:
> do we have a variable in kitchen telling us whether the test is for a custom ami or not

yes, we have. `node['cfncluster']['is_official_ami_build']` indicates it is official ami not custom ami.